just_audio_platform_interface/lib/just_audio_platform_interface.dart

```diff
@@ -1055,15 +1135,18 @@ class ProgressiveAudioSourceMessage extends UriAudioSourceMessage {
     required String id,
     required String uri,
     Map<String, String>? headers,
+    List<AudioEffectMessage>? effects,
     dynamic tag,
-  }) : super(id: id, uri: uri, headers: headers, tag: tag);
+  }) : super(id: id, uri: uri, headers: headers, tag: tag, effects: effects);
 
   @override
   Map<dynamic, dynamic> toMap() => <dynamic, dynamic>{
         'type': 'progressive',
         'id': id,
         'uri': uri,
         'headers': headers,
+        'effects':
+            effects?.map((audioEffectMessage) => audioEffectMessage.toMap())
```

Comment:
Nota: Dovunque lo hai gli effetti
```suggestion
        'effects': effects?.map((audioEffectMessage) {
          return audioEffectMessage.toMap();
        }).toList(),
```